### PR TITLE
khatm

### PR DESCRIPTION
**Pull request by ARHariri**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -31,6 +31,8 @@ import {CreateKhatmPage} from "../pages/create-khatm/create-khatm";
 import {LanguageService} from "../services/language";
 import {KhatmService} from "../services/khatm.service";
 import {SocialSharing} from "@ionic-native/social-sharing";
+import {Network} from "@ionic-native/network";
```
Comment (aminazar) [inline src/app/app.module.ts]: do we use this import in this file?

File: src/components/left-menu/left-menu.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
         </ion-select>
       </ion-item>
     <button ion-item detail-none (click)="nightMode()">
-      <ion-icon [name]="stylingService.nightMode?'eye':'eye-off'" item-left></ion-icon>
+      <ion-icon [name]="stylingService.nightMode ? 'sunny' : 'moon'" item-left></ion-icon>
```
Comment (aminazar) [inline src/components/left-menu/left-menu.html:20]: good replacement :)

File: src/components/right-menu/right-menu.html [hunk 3]
```diff
@@ -1,21 +1,35 @@
...
       <ion-row>
         <ion-col col-6>
-          <button ion-button menuClose *ngIf="!isLoggedIn" block (click)="openPage('register')">{{ls.translate('Register')}}</button>
-          <button ion-button menuClose *ngIf="isLoggedIn" block (click)="logout()">{{ls.translate('Logout')}}</button>
+          <button [color]="conditionalColoring.primary" ion-button menuClose *ngIf="!isLoggedIn" block (click)="openPage('register')">{{ls.translate('Register')}}</button>
+          <button [color]="conditionalColoring.primary" ion-button menuClose *ngIf="isLoggedIn" block (click)="logout()">{{ls.translate('Logout')}}</button>
         </ion-col>
         <ion-col col-6>
-          <button ion-button menuClose *ngIf="isLoggedIn" block (click)="openPage('khatm')">{{ls.translate('Create Khatm')}}</button>
+          <button [color]="conditionalColoring.primary" ion-button menuClose *ngIf="isLoggedIn" block (click)="openPage('khatm')">{{ls.translate('Create Khatm')}}</button>
```
Comment (aminazar) [inline src/components/right-menu/right-menu.html:10]: great - thanks for fixing background colour issue for side menus

File: src/services/language.ts [hunk 4]
```diff
@@ -140,7 +140,8 @@ export class LanguageService {
 
   convertDate(date){
     if(this._lang === 'fa'){
-      momentJalali.loadPersian(true);
+      // momentJalali.loadPersian(true);
+      momentJalali.loadPersian();
```
Comment (aminazar) [inline src/services/language.ts]: it did not work without 'true' parameter for me or Ms Salehi

**Review by aminazar — COMMENTED** [review]
Thanks - it seems you finished commitment page.
Did you upload the app to ionic after that?

File: src/app/app.module.ts (hunk 1 above)
Comment (ARHariri) [reply src/app/app.module.ts]: Yes. When remove it from the code, we have an error: 'No provider for network'

File: src/services/language.ts (hunk 4 above)
Comment (ARHariri) [reply src/services/language.ts]: It is strange. It did not work with 'true' parameter for me. For uploading on ionic server I should remove the 'true' parameter.

Comment (ARHariri) [reply src/services/language.ts]: Sorry for late. Resolve this issue. I remove 'jalali-moment' package and install it again. This function need one Boolean parameter.

**Pull request merged**